tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -168,11 +168,35 @@ def check_meshcop_service_by_data(self, br, service_data):
         logging.info(bin(state_bitmap))
         self.assertEqual((state_bitmap & 7), 1)  # connection mode = PskC
         if br.get_state() == 'disabled':
-            self.assertEqual((state_bitmap >> 3 & 3), 0)  # Thread is disabled
+            self.assertEqual(
+                (state_bitmap >> 3 & 3), 0
+            )  # Thread interface is not active and is not initialized with a set of valid operation network parameters
+            self.assertEqual((state_bitmap >> 9 & 3), 0)  # Thread role is disabled
```

Comment:
- You may extract the bits before the if clauses so you don't have to repeat `(state_bitmap >> 3 & 3)`.
```python
interface_status = state_bitmap >> 3 & 3
role = state_bitmap >> 9 & 3
if br.get_state() == ...:
```
- For the long comment, probably it's more clear to put it above the assertion so that the assertion won't be split into multiple lines by the formatter.

Reply:
updated as your suggestion.